imod/tests/test_mf6/test_mf6_npf.py

```diff
@@ -208,3 +210,94 @@ def test_configure_xt3d(tmp_path):
     assert "xt3d" not in rendered
     assert "rhs" not in rendered
     assert not npf.get_xt3d_option()
+
+
+@pytest.mark.usefixtures("imod5_dataset")
+def test_npf_from_imod5_isotropic(imod5_dataset, tmp_path):
+    data = deepcopy(imod5_dataset[0])
+    # throw out kva (=vertical anisotropy array) and ani (=horizontal anisotropy array)
+    data.pop("kva")
+    data.pop("ani")
+
+    _load_imod5_data_in_memory(data)
+    target_grid = data["khv"]["kh"]
+    npf = imod.mf6.NodePropertyFlow.from_imod5_data(data, target_grid)
+
+    # Test array values are the same  for k ( disregarding the locations where k == np.nan)
+    k_nan_removed = xr.where(np.isnan(npf.dataset["k"]), 0, npf.dataset["k"])
+    np.testing.assert_allclose(k_nan_removed, data["khv"]["kh"].values)
+
+    rendered_npf = npf.render(tmp_path, "npf", None, None)
+    assert "k22" not in rendered_npf
+    assert "k33" not in rendered_npf
+    assert "angle1" not in rendered_npf
+    assert "angle2" not in rendered_npf
+    assert "angle3" not in rendered_npf
+
+
+@pytest.mark.usefixtures("imod5_dataset")
+def test_npf_from_imod5_horizontal_anisotropy(imod5_dataset, tmp_path):
+    data = deepcopy(imod5_dataset[0])
+    # throw out kva (=vertical anisotropy array)
+    data.pop("kva")
+
+    _load_imod5_data_in_memory(data)
+    target_grid = data["khv"]["kh"]
+    data["ani"]["angle"].values[:, :, :] = 135.0
+    data["ani"]["factor"].values[:, :, :] = 0.1
+    npf = imod.mf6.NodePropertyFlow.from_imod5_data(data, target_grid)
+
+    # Test array values  for k22 and angle1
+    for layer in npf.dataset["k"].coords["layer"].values:
+        k_layer = npf.dataset["k"].sel({"layer": layer})
+        k22_layer = npf.dataset["k22"].sel({"layer": layer})
+        angle1_layer = npf.dataset["angle1"].sel({"layer": layer})
```

Comment:
Tip: You can select complete datasets in one go, saves some code:

```suggestion
        ds_layer = npf.dataset.sel({"layer": layer})
```

Reply:
done